### PR TITLE
Fix a bug in Extra Utilities that caused fluid to be voided when using fluid retrieval nodes

### PR DESCRIPTION
**Pull request by FourIsTheNumber**
Addresses this issue: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16324#issue-2306716093

Added a new mixin to overwrite "transfer" method in FluidBufferRetrieval class in ExU to fix this bug.

This is my first contribution and first time working with mixins, happy to fix any mistakes I almost certainly have made!

### CODE REVIEW

File: src/main/resources/mixins.hodgepodge.late.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
   "package": "com.mitchej123.hodgepodge.mixins.late",
   "refmap": "mixins.hodgepodge.refmap.json",
   "target": "@env(DEFAULT)",
-  "compatibilityLevel": "JAVA_8"
+  "compatibilityLevel": "JAVA_8",
+  "mixins": [
+    "extrautilities.MixinFluidBufferRetrieval"
+  ]
```
Comment (OneEyeMaker) [inline src/main/resources/mixins.hodgepodge.late.json:10]: There's no need to manually register mixin classes. These entries are dynamically generates from classes listed in https://github.com/GTNewHorizons/Hodgepodge/blob/master/src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

Comment (FourIsTheNumber) [reply src/main/resources/mixins.hodgepodge.late.json:10]: Makes total sense. Mixins.java was one of the last things I found, embarrassingly, so I had already registered it manually. Thanks!

Comment (OneEyeMaker) [reply src/main/resources/mixins.hodgepodge.late.json:10]: Then please, revert changes in JSON file (remove these unnecessary lines).

Comment (FourIsTheNumber) [reply src/main/resources/mixins.hodgepodge.late.json]: Done, sorry about that

**Review by OneEyeMaker — APPROVED** [review]

**Comment by OneEyeMaker** [comment]
@FourIsTheNumber 
And no need to be so embarrassed or sorry when reviewer points at an issue. We *all* do mistakes without exceptions. So don't worry about that so much.

Anyway, *congratulations on your first contribution to the GTNH project!* 🎆

**Pull request merged**